src/app/CommandHandler.cpp

```diff
@@ -97,6 +98,60 @@ void CommandHandler::OnInvokeCommandRequest(Messaging::ExchangeContext * ec, con
     mGoneAsync = true;
 }
 
+CHIP_ERROR CommandHandler::ValidateCommands(TLV::TLVReader & invokeRequestsReader)
```

Comment:
Suggest making ValidateCommands a public method, so that ti can be unit-tested on its own.